packages/desktop-client/src/components/modals/ImportTransactionsModal/ParsedDate.tsx

```diff
@@ -27,13 +28,13 @@ export function ParsedDate({
       <Text>
         {date || (
           <Text style={{ color: theme.pageTextLight, fontStyle: 'italic' }}>
-            Empty
+            <Trans>Empty</Trans>
```

Comment:
It would maybe be nice to add context on what is empty here via `<Trans context="..."`. But ff to defer to another PR